wallet/src/main/java/ch/admin/bag/covidcertificate/wallet/pdf/PdfViewModel.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2021 Ubique Innovation AG <https://www.ubique.ch>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package ch.admin.bag.covidcertificate.wallet.pdf
+
+import android.app.Application
+import android.content.ClipData
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import ch.admin.bag.covidcertificate.common.qr.QRCodeReaderHelper
+import ch.admin.bag.covidcertificate.eval.data.state.DecodeState
+import ch.admin.bag.covidcertificate.eval.decoder.CertificateDecoder
+import java.io.File
+import java.io.InputStream
+import ch.admin.bag.covidcertificate.eval.data.state.Error
+
+class PdfViewModel(application: Application) : AndroidViewModel(application) {
+
+	private val pdfImportMutableLiveData: MutableLiveData<DecodeState> = MutableLiveData()
+	val pdfImportLiveData: LiveData<DecodeState> = pdfImportMutableLiveData
+
+	fun importPDFData(clipData: ClipData) {
```

Comment:
For consistency we could name the functions `importPdfData` (or even `importPdf`) and `cleanPdf` (or `clearPdf`?).